style.css

```diff
@@ -1,564 +1,563 @@
+  height: 0.1rem;
+  width: 3.2rem;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: -1rem;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.feature-boxes {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.feature-box {
+  background-color: rgba(206, 212, 218, 0.1);
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+  width: 100%;
+  padding: 1rem;
+}
+
+.feature-logo {
+  width: 4.8rem;
+}
+
+.feature-title {
+  color: var(--main-color);
+  font-size: 1.2rem;
+  font-weight: 400;
+  width: 20%;
+}
+
+.feature-desc {
+  color: #fff;
+  font-size: 1.2rem;
+  justify-self: end;
+  width: 60%;
+}
+
+.feature-btn1 {
+  padding: 2rem;
+  color: #fff;
+  background-color: var(--main-color);
+  border: none;
+  margin: 1.8rem 0;
+  cursor: pointer;
+}
+
+.feature-btn1:hover {
+  background-color: #a5392e;
+}
+
+.feature-btn2 {
+  display: none;
+}
+
+/* ***************** */
+
+/* **** SPEAKER ***** */
+
+/* ***************** */
+.speaker {
+  padding: 1.8rem 1.4rem;
+}
+
+.speaker-title {
+  color: #000;
+  font-size: 2.4rem;
+  font-weight: 400;
+  position: relative;
+  margin-bottom: 3.2rem;
+  text-align: center;
+}
+
+.speaker-title::after {
+  content: '';
+  height: 0.1rem;
+  width: 3.2rem;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: -1rem;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.speaker-box {
+  display: flex;
+  gap: 1.8rem;
+  margin-bottom: 1.8rem;
+}
+
+.speaker-img {
+  width: 11rem;
+}
+
+.speaker-details {
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+}
+
+.speaker-name {
+  font-size: 1.8rem;
+}
+
+.speaker-position {
+  font-size: 1.2rem;
+  color: var(--main-color);
+  position: relative;
+  margin-bottom: 1rem;
+  font-style: italic;
+}
+
+.speaker-position::after {
+  content: '';
+  height: 0.1rem;
+  width: 2rem;
+  background-color: #868e96;
+  position: absolute;
+  bottom: -1rem;
+  left: 0;
+}
+
+.speaker-desc {
+  line-height: 1.5;
+}
+
+/* ***************** */
+
+/* **** PARTNERS ***** */
+
+/* ***************** */
+
+.partner {
+  background-color: #272a31;
+  padding: 3.2rem;
+  display: none;
+}
+
+.partner-title {
+  color: #fff;
+  font-size: 2.4rem;
+  font-weight: 400;
+  position: relative;
+  margin-bottom: 3.2rem;
+  text-align: center;
+}
+
+.partner-title::after {
+  content: '';
+  height: 0.1rem;
+  width: 3.2rem;
+  background-color: var(--main-color);
+  position: absolute;
+  bottom: -1rem;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.partners {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  flex-wrap: wrap;
+}
+
+.partners img {
+  height: 4rem;
+  padding: 0.5rem;
+  filter: brightness(0);
+  opacity: 50%;
+}
+
+/* ***************** */
+
+/* **** FOOTER ***** */
+
+/* ***************** */
+.footer {
+  padding: 2rem 3rem;
+  display: flex;
+  justify-content: space-between;
+  gap: 2.4rem;
+}
+
+.footer img {
+  height: 2.4rem;
+}
```

Comment:
Kindly update the size of the logo in the footer for your about page. It is causing a horizontal scroll on mobile view as seen in the screenshot below:

<img width="238" alt="image" src="https://github.com/kkmanuu/Capstone-1/assets/37116322/0a443f78-1a2b-4e69-9ea3-bc52e0da8378">